index.js

```diff
@@ -0,0 +1,70 @@
+const button = document.getElementById("button")
+const taskList = document.getElementById("taskList")
+let progressList = document.getElementsByName('process')
+let newTask = document.getElementById("newTask")
+let inicialTableTag = 
+'<tr>' +
+    '<th>' + "ID" + '</th>' +
+    '<th>' + "コメント" + '</th>' +
+    '<th>' + "状態" + '</th>' +
+'<tr>'
+taskList.innerHTML = inicialTableTag
+const tasks = [];
+
+//新規タスクの追加ボタンが押下された時の処理
+button.addEventListener("click", ()=>{
+    if(newTask.value == ""){
```

Comment:
`==` ではなく `===` を使うようにして下さい

```suggestion
    if(newTask.value === ""){
```

Reply:
修正しました。